Formula/code-server.rb

```diff
@@ -60,6 +60,6 @@ def plist
 
   test do
     system bin/"code-server", "--extensions-dir=.", "--install-extension", "ms-python.python"
-    assert_equal "ms-python.python\n", shell_output("#{bin/"code-server"} --extensions-dir=. --list-extensions")
+    assert_equal "info  Using config file ~/.config/code-server/config.yaml\nms-python.python\n", shell_output("#{bin/"code-server"} --extensions-dir=. --list-extensions")
```

Comment:
If you split this onto two lines, the line length audit will be happier.

```suggestion
    assert_equal "info  Using config file ~/.config/code-server/config.yaml\nms-python.python\n",
      shell_output("#{bin/"code-server"} --extensions-dir=. --list-extensions")
```